tests/stdlib/test_math.py

```diff
@@ -8,4 +9,46 @@ def test_sqrt(self):
         self.assertCodeExecution("""
             import math
             print(math.sqrt(1))
-            """)
+        """)
+
+    def test_exp_standard(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.exp(0))
+            print(math.exp(-1))
+            print(math.exp(1))
+            print(math.exp(0.0001))
+            print(math.exp(-0.0001))
+            print(math.exp(-100000))
+        """)
+
+    @expectedFailure
+    def test_exp_fail_empty(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.exp())
+        """)
+
+    @expectedFailure
+    def test_exp_fail_nan(self):
+        self.assertCodeExecution("""
```

Comment:
`import math` here?
the test fails because math.exp() can't be found?

Reply:
done :)